hubspot3/tickets.py

```diff
@@ -80,19 +80,27 @@ def get(self, ticket_id: str, include_deleted: bool = False, **options) -> Dict:
             "objects/tickets/{}".format(ticket_id), method="GET", **options
         )
 
-    def _join_output_properties(self, tickets: List[dict]) -> List[dict]:
+    def _join_output_properties(self, tickets: List[dict]) -> dict:
         """
-        Join request properties to show only one output per ticketId
+        Join request properties to show only one object per ticketId
         This will change the first object for each ticketId
         """
         joined_tickets_dict = {}
         for ticket in tickets:
-            ticket_id = ticket["objectId"]
+            # Converting the ID to str to make it compatible with API
+            ticket_id = str(ticket["objectId"])
             if ticket_id not in joined_tickets_dict:
                 joined_tickets_dict[ticket_id] = ticket
             else:
                 joined_tickets_dict[ticket_id]["properties"].update(ticket["properties"])
-        return list(joined_tickets_dict.values())
+        return joined_tickets_dict
+
+    def _join_output_properties_to_list(self, tickets: List[dict]) -> List[dict]:
+        """
+        Create a list with joined request properties to show only one object per ticketId
+        This will change the first object for each ticketId
+        """
+        return list(self._join_output_properties(tickets).values())
 
     def _split_properties(self, properties: Set[str],
                           max_properties_request_length: int = None,
```

Comment:
`int` cannot be `None`. The type here should be `max_properties_request_length: Optional[int] = None`.